src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -95,6 +96,16 @@ List<Message> messagesByUser = (List<Message>) request.getAttribute("messagesByU
       </div>
       <hr/>
     <% } %>
+
+    <h1>Profile pages with "hashtag"</h1>
+    <ul>
+      <% for (User user: users) { %>
+        <% if (user.getAboutMe().toLowerCase().contains("hashtag")) { %>
+            <li><a href="/users/<%= user.getAboutMe() %>">
+            <%= user.getAboutMe()%></a></li>
+        <% } %>
```

Comment:
*104*: Here, I'm working on configuring a way for the `href`` tag to link each description to the author's page in which the content was published